src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -150,7 +150,7 @@ class Comment extends React.Component<Props, State> {
                                             data-testid="edit-comment"
                                             onClick={this.handleEditClick}
                                         >
-                                            <IconPencil color={bdlGray80} />
+                                            <Pencil16 color={bdlGray} />
```

Comment:
the new icons do not have a color prop, so not sure if this is working?

Reply:
Right. Just dropped the color prop. The default color for both these icons is `bdlGray` which is what we want.